Evaluation/Input/evaluation_plan.json

```diff
@@ -228,28 +231,12 @@
                                 "Project": "Alfentanil",
                                 "Simulation": "Kharasch 2011b, Alfentanil IV 1 mg",
                                 "Output": "Organism|PeripheralVenousBlood|Alfentanil|Plasma (Peripheral Venous Blood)",
-                                "ObservedData": "Kharasch 2011b - Alfentanil IV Control sequential - Alfentanil - IV - 1 mg - Plasma - agg. (n=6)",
-                                "Color": "#FF0000"
-                            },
-                            {
-                                "Project": "Alfentanil",
-                                "Simulation": "Kharasch 2011b, Alfentanil IV 1 mg",
-                                "Output": "Organism|PeripheralVenousBlood|Alfentanil|Plasma (Peripheral Venous Blood)",
-                                "ObservedData": "Kharasch 2011b - Alfentanil IV Control simultaneous - Alfentanil - IV - 1 mg - Plasma - agg. (n=6)",
-                                "Color": "#FF0000"
-                            },
-                            {
-                                "Project": "Alfentanil",
-                                "Simulation": "Kharasch 2011b, Alfentanil IV 1 mg",
-                                "Output": "Organism|PeripheralVenousBlood|Alfentanil|Plasma (Peripheral Venous Blood)",
-                                "ObservedData": "Kharasch 2011b - Alfentanil IV Grapefruit coadmin sequential - Alfentanil - IV - 1 mg - Plasma - agg. (n=6)",
-                                "Color": "#FF0000"
-                            },
-                            {
-                                "Project": "Alfentanil",
-                                "Simulation": "Kharasch 2011b, Alfentanil IV 1 mg",
-                                "Output": "Organism|PeripheralVenousBlood|Alfentanil|Plasma (Peripheral Venous Blood)",
-                                "ObservedData": "Kharasch 2011b - Alfentanil IV Grapefruit coadmin simultaneous - Alfentanil - IV - 1 mg - Plasma - agg. (n=6)",
+                                "ObservedData": [
+                                    "Kharasch 2011b - Alfentanil IV Control sequential - Alfentanil - IV - 1 mg - Plasma - agg. (n=6)",
+                                    "Kharasch 2011b - Alfentanil IV Control simultaneous - Alfentanil - IV - 1 mg - Plasma - agg. (n=6)",
+                                    "Kharasch 2011b - Alfentanil IV Grapefruit coadmin sequential - Alfentanil - IV - 1 mg - Plasma - agg. (n=6)",
+                                    "Kharasch 2011b - Alfentanil IV Grapefruit coadmin simultaneous - Alfentanil - IV - 1 mg - Plasma - agg. (n=6)"
+                                ],
```

Comment:
use array of observed data instead of duplication.
Can be used only if all other properties (project, simulation, output, color,...) are the same - which was the case here